plugin/al/plugin/AL_USDMayaTestPlugin/py/testTranslators.py

```diff
@@ -417,6 +458,20 @@ def test_deletion_of_parent_node_by_translator_does_not_crash_Maya(self):
 
         vs.SetVariantSelection("noCubes")
 
+    def test_import_and_update_consistency(self):
+        '''
+        test consistency when called via TranslatePrim, or triggered via onObjectsChanged
+        '''
+
+        usdmaya.TranslatorBase.registerTranslator(UpdateableTranslator(), 'test')
+
+        stage = Usd.Stage.Open("../test_data/translator_update_postimport.usda")
+        stageCache = UsdUtils.StageCache.Get()
+        stageCache.Insert(stage)
+        stageId = stageCache.GetId(stage)
+        shapeName = 'updateProxyShape'
+        cmds.AL_usdmaya_ProxyShapeImport(stageId=stageId.ToLongInt(), name=shapeName)
```

Comment:
How does this test work? I'm not sure I understand how we detect errors with this test? I don't see anything thrown or collected by the UpdateableTranslator.